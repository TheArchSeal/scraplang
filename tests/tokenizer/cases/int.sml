1
12
1_2_3_
0b1
0b1100
0b_111_10_1_1_
0x1
0xC
0x_7b_
