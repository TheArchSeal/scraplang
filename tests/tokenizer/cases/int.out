(2):1:1 1 1
(2):2:1 12 12
(2):3:1 1_2_3_ 123
(2):4:1 0b1 1
(2):5:1 0b1100 12
(2):6:1 0b_111_10_1_1_ 123
(2):7:1 0x1 1
(2):8:1 0xC 12
(2):9:1 0x_7b_ 123
